B-Minus-Programmers/jojibrush
# Shape Properties

Closes #38

Implements properties for Geometric and Text shapes, defined in Properties.hpp. The base Shape class automatically calls applyProperties() on the QPainter when any shape is drawn. The applyProperties() method is a virtual method defined in GeometricShape and Text, which applies the relevant struct of properties to the QPainter.

Test:
 - Open the project, make sure that the square is colorful and dashed
 - Make your own shapes in mainwindow.cpp if you want to play :)
